include/attribute.h

```diff
@@ -3,6 +3,11 @@
 
 #include "global.h"
 
+/* Array index defines. Used in itemQuantityList */
+#define NUM_WOUND_BAND 0
+#define NUM_JVHUA_CHA 1
+#define GOOD_POTION 2
```

Comment:
missing `NUM_`? And I think `ITEM_QUANTITY_xxx_IDX` is better than `NUM_xxx`, which is not clear.